keploy.sh

```diff
@@ -1,184 +1,223 @@
 #!/bin/bash
 
-installKeploy (){
-    IS_CI=false
-    for arg in "$@"
-    do
-        case $arg in
-            -isCI)
-                IS_CI=true
-                shift
+# Function to display the menu and handle user selection
+platform="Linux"
+OS_NAME="$(uname -s)"
+IS_CI=false
+
+isRunningOnWindows (){
+    if [[ "$OS_NAME" == MINGW32_NT* ]] || [[ "$OS_NAME" == MINGW64_NT* ]]; then
```

Comment:
this isn't just a windows testing rather a OS detection function. Update accordingly